pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -858,7 +858,7 @@ benchmarks! {
 
 	round_transition_on_initialize {
 		// TOTAL SELECTED COLLATORS PER ROUND
-		let x in 1..28;
+		let x in 8..40;
```

Comment:
What is the reason behind this being 40?

Reply:
We should probably test well beyond what we expect to support. I say this mainly because we want to ensure that we capture any scaling characteristics that are far worse than linear.

Imagine benchmarking the fn `f(x) { x*x }` and producing a linear fit to it. If you only benchmark very low inputs, you will have a decent fit with those numbers but will vastly underestimate large inputs. On the other hand, if you benchmark large inputs you will overstate low inputs and have a better fit for large inputs.

The latter is safer at the expense of being expensive, but it also will yield numbers that hint that the underlying algorithms having poor scaling characteristics.

Note that I'm not suggesting that this is the case (scaling significantly worse than `O(n)`), but that this is generally a good defensive approach to benchmarking.